### PR TITLE
refactor: use more structured bindings

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
We should use structured bindings where possible

## What was done?
Use them

## How Has This Been Tested?
Compiled

## Breaking Changes
None

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by knst — APPROVED** [review]
utACK 7f1f660b37157536ae2af7586316fd2a384b46e4

File: src/llmq/dkgsessionhandler.cpp [hunk 1]
```diff
@@ -367,24 +367,22 @@ std::set<NodeId> BatchVerifyMessageSigs(CDKGSession& session, const std::vector<
     pubKeys.reserve(messages.size());
     messageHashes.reserve(messages.size());
     bool first = true;
-    for (const auto& p : messages ) {
-        const auto& msg = *p.second;
-
-        auto member = session.GetMember(msg.proTxHash);
+    for (const auto& [nodeId, msg] : messages ) {
```
Comment (UdjinM6) [inline src/llmq/dkgsessionhandler.cpp:370]: clang format complains:
```suggestion
    for (const auto& [nodeId, msg] : messages) {
```

**Review by UdjinM6 — COMMENTED** [review]
LGTM, one tiny issue

File: src/llmq/dkgsessionhandler.cpp (hunk 1 above)
Comment (PastaPastaPasta) [reply src/llmq/dkgsessionhandler.cpp]: done

**Review by UdjinM6 — APPROVED** [review]
utACK fb0b5ac85940e37a5e53aa507bb9bd8a80a071a0

**Pull request merged**